expo/gbe/functions.py

```diff
@@ -145,3 +146,24 @@ def available_volunteers(event_start_time):
         if event_start_time in window_range:
             windows.append(window)
     return conf.Volunteer.objects.filter(available_windows__in=windows)
+
+
+def get_events_and_windows(conference):
```

Comment:
I disagree - this function also queries a table in the gbe app, so it should be in the gbe app, no?

My assertion - scheduler should not have to know about gbe since scheduler should work for other forms of conference logic.
I don't know that I can make the same assertion for gbe.